src/middlewared/middlewared/plugins/cloud_sync.py

```diff
@@ -175,7 +175,7 @@ async def rclone(middleware, job, cloud_sync):
                 await middleware.call("zfs.snapshot.create", dict(snapshot, recursive=recursive))
 
                 relpath = os.path.relpath(path, dataset["mountpoint"])
```

Comment:
According to flake8 this isnt used anymore? did you mean to put relpath after snapshot_name below?